### PR TITLE
Add widget mappings for ALL spellbooks (Standard, Ancient, Lunar, and Arceuus)

### PR DESCRIPTION
**Pull request by DannysPVM**
This PR adds mapping to all widgets in all spellbooks (Standard, Ancient, Lunar, and Arceuus).

<details>
<summary>Screenshots of mapped widgets in the widget inspector</summary>

Standard Spellbook |
--------------------|
image | 

Ancient Spellbook |
-------------------|
image | 

Lunar Spellbook |
-----------------|
image | 

Arceuus Spellbook |
-------------------|
image

</details>

### CODE REVIEW

**Comment by deathbeam** [comment]
Needs rebase

**Comment by DannysPVM** [comment]
Rebased, thanks Deathbeam.

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java [hunk 1]
```diff
@@ -108,6 +108,7 @@
 	public static final int KILL_LOGS_GROUP_ID = 549;
 	public static final int DIARY_QUEST_GROUP_ID = 275;
 	public static final int THEATRE_OF_BLOOD_GROUP_ID = 23;
+  	public static final int SPELLBOOK_GROUP_ID = 218;
```
Comment (deathbeam) [inline runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java]: Fix this indentation

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java [hunk 2]
```diff
@@ -590,4 +591,200 @@
 		static final int KILLS = 14;
 		static final int STREAK = 15;
 	}
-}
\ No newline at end of file
+  
+  	static class StandardSpellBook
```
Comment (deathbeam) [inline runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java]: fix this indentation

**Comment by deathbeam** [comment]
looks like travis is failing

**Pull request closed without merging**